test/xpu/run_test.py

```diff
@@ -0,0 +1,369 @@
+    "test_variant_consistency_eager_jiterator_unary_xpu_complex64",
+    "test_variant_consistency_eager_jiterator_unary_xpu_float32",
+    "test_variant_consistency_eager_nn_functional_conv1d_xpu_complex64",
+    "test_variant_consistency_eager_nn_functional_conv1d_xpu_float32",
+    "test_variant_consistency_eager_nn_functional_conv2d_xpu_complex64",
+    "test_variant_consistency_eager_nn_functional_conv2d_xpu_float32",
+    "test_variant_consistency_eager_nn_functional_conv3d_xpu_complex64",
+    "test_variant_consistency_eager_nn_functional_conv3d_xpu_float32",
+    "test_variant_consistency_eager_nn_functional_conv_transpose1d_xpu_complex64",
+    "test_variant_consistency_eager_nn_functional_conv_transpose1d_xpu_float32",
+    "test_variant_consistency_eager_nn_functional_conv_transpose2d_xpu_complex64",
+    "test_variant_consistency_eager_nn_functional_conv_transpose2d_xpu_float32",
+    "test_variant_consistency_eager_nn_functional_conv_transpose3d_xpu_complex64",
+    "test_variant_consistency_eager_nn_functional_conv_transpose3d_xpu_float32",
+    "test_variant_consistency_eager_nn_functional_group_norm_xpu_float32",
+    "test_variant_consistency_eager_nn_functional_rrelu_xpu_float32",
+    "test_variant_consistency_eager_to_sparse_xpu_complex64",
+    "test_variant_consistency_eager_to_sparse_xpu_float32",
+    "test_compare_cpu__native_batch_norm_legit_xpu_float32",
+    "test_compare_cpu__refs_special_zeta_xpu_float32",
+    "test_compare_cpu_linalg_lu_factor_ex_xpu_float32",
+    "test_compare_cpu_linalg_lu_factor_xpu_float32",
+    "test_compare_cpu_linalg_lu_xpu_float32",
+    "test_compare_cpu_native_batch_norm_xpu_float32",
+    "test_compare_cpu_special_hermite_polynomial_h_xpu_float32",
+    "test_compare_cpu_special_zeta_xpu_float32",
+    "test_out_cholesky_inverse_xpu_float32",
+    "test_out_geqrf_xpu_float32",
+    "test_out_narrow_copy_xpu_float32",
+    "test_out_ormqr_xpu_float32",
+    "test_out_triangular_solve_xpu_float32",
+    "test_python_ref__refs_heaviside_xpu_int64",
+    "test_python_ref__refs_special_bessel_j0_xpu_int64",
+    "test_python_ref_errors__refs_dstack_xpu",
+    "test_python_ref_errors__refs_hstack_xpu",
+    "test_python_ref_errors__refs_linalg_cross_xpu",
+    "test_python_ref_errors__refs_masked_fill_xpu",
+    "test_python_ref_errors__refs_vstack_xpu",
+    "test_python_ref_executor__refs_mul_executor_aten_xpu_complex32",
+    "test_python_ref__refs_special_multigammaln_mvlgamma_p_5_xpu_float64",
+    "test_python_ref_executor__refs_minimum_executor_aten_xpu_int64",
+    "test_python_ref_executor__refs_special_multigammaln_mvlgamma_p_3_executor_aten_xpu_float64",
+)
+
+
+skip_options = " -k 'not " + skip_list[0]
+for skip_case in skip_list[1:]:
+    skip_option = " and not " + skip_case
+    skip_options += skip_option
+skip_options += "'"
+
+test_command = "PYTORCH_ENABLE_XPU_FALLBACK=1 PYTORCH_TEST_WITH_SLOW=1 pytest -v _test_ops.py"
```

Comment:
Will we have other `_test_xxx.py` under this folder? if have, do we need to define multiple skip lists for those test suites respectively? And we will have multiple test commands in here, right?

Reply:
Ideally, I believe no, since each case gets unique test name.

But so far, `pytest -v` looks not working for the ported cases. We need use `pytest -v _test_xxx.py`. I believe it should work  with some change in patching scheme.

Reply:
We can optimize validation script in another task thread.